internal/service/advancedclustertpf/model_to_ClusterDescription20240805.go

```diff
@@ -61,7 +61,7 @@ func newBiConnector(ctx context.Context, input types.Object, diags *diag.Diagnos
 	}
 }
 func newComponentLabel(ctx context.Context, input types.Set, diags *diag.Diagnostics) *[]admin.ComponentLabel {
-	if input.IsUnknown() || input.IsNull() {
+	if input.IsUnknown() {
```

Comment:
using blocks (SetNestedBlock), a default value can't be set.  So we return an empty set (instead of nil) when input is nil because it's not being provided.  same for labels and tags